addons/escoria-core/game/project_settings_manager.gd

```diff
@@ -0,0 +1,123 @@
+# Registers and allows access to Escoria-specific project settings.
+extends Node
+class_name ESCProjectSettingsManager
+
+
+# Root for Escoria-specific project settings
+const _ESCORIA_SETTINGS_ROOT = "escoria"
+
+# UI-related Escoria project settings
+const _UI_ROOT = "ui"
+
+const DEFAULT_DIALOG_TYPE = "%s/%s/default_dialog_type" % [_ESCORIA_SETTINGS_ROOT, _UI_ROOT]
```

Comment:
Should we add the settings descriptions here as well? That would make it easy to document it, because this file would be API-documented anyway.

(Doesn't need to be in this PR, we can add them in a separate issue)

Reply:
Makes sense to me. I agree re adding a new issue.